tests/test_data_types.py

```diff
@@ -0,0 +1,36 @@
+import json
+import pathlib
+from unittest import TestCase
+
+from tecton_client import GetFeaturesResponse
+
+TEST_DATA_DIR = pathlib.Path(__file__).parent.joinpath("test_data")
+
+
+class TestDataTypes(TestCase):
+    def test_GetFeaturesResponse_from_dict(self):
+        self.maxDiff = 10000
+        file_1 = TEST_DATA_DIR.joinpath("sample_response.json")
+        with open(file_1) as f:
+            resp = json.load(f)
+        resp = GetFeaturesResponse.from_dict(resp)
+        self.assertEquals(resp.result.features, [["0"], None, [55.5, 57.88, 58.96, 57.66, None, 55.98]])
```

Comment:
Hmm does our client _not_ parse the int64 we have encoded as a string to a Python number? That seems like something our client should do right?

Reply:
I don't think we necessarily know here if this is an array of strings vs an array of int64; The client may not have requested the DataTypes.

I think having another method, `GetFeaturesResponse.get_features()`, which constructs a friendlier representation of the features, assuming the user has requested both the names and datatypes would make sense, and I was thinking of doing a follow-up PR with that. We could do decoding there.

Reply:
> I think having another method, GetFeaturesResponse.get_features(), which constructs a friendlier representation of the features, assuming the user has requested both the names and datatypes would make sense, and I was thinking of doing a follow-up PR with that

something in this vein sounds like a good idea + follow up 👍

Reply:
I forgot that our client isn't _requiring_ datatypes/fieldname metadata